app/api/v1/router.py

```diff
@@ -20,3 +21,4 @@
     prefix="/users/{user_id}/comprehensive_evaluations",
     tags=["comprehensive_evaluations"],
 )
+router.include_router(progress.router, prefix="/progresses", tags=["Progress"])
```

Comment:
@Ryan-OH 
```suggestion
router.include_router(progress.router, prefix="/ users/{user_id}/progress", tags=["Progress"])
```
여기 user 가 상위 테이블이니까로 위처럼 하시는건 어떠신가요 그럼 request body에 유저는 따로 안받아도 될 거 같은뎅

Reply:
아 그러면 되겠네요 좋은 의견 감사합니다